mentors/python/v.json

```
@@ -22,5 +22,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I'm a medical student who programs as a hobby. I love python for it's elegance and readability; it was my first langauge and it remains my favorite."
-  }
+  },
+  {
+    "github_username": "vidhisshah",
+    "name": "Vidhi",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I've been coding for a deacde. Out all the languages I have learnt and used, Python is my favourite."
```

Small grammar changes:
```suggestion
    "bio": "I've been coding for a decade. Out of all the languages I have learnt and used, Python is my favourite."
```